File: solutions/0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] n, int target) {
        int sum=0,k=0;
        int arr[]=new int[2];
        for(int i=0;i<n.length;i++){
            sum=0;
            for(int j=i+1;j<n.length;j++){
                sum=n[i]+n[j];
                if(sum==target){
                    arr[k]=i;
                    arr[++k]=j;
                    //return arr;
                    break;
                }
            }
        }
        return arr;
    }
}

File: solutions/0001-two-sum/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int>ans;
        for(int i=0;i<nums.size();i++){
            int rm=target-nums[i];
            if(ans.find(rm)!=ans.end()){
                return {ans[rm],i};
            }
            ans[nums[i]]=i;
        }
        return {};
    }
};
